### DISCUSSION TITLE
graphscope是否集成networkx的所有算法？

### DISCUSSION THREAD

**Discussion by doingGod** [alibaba/GraphScope#1988] [Q&A]
您好，
我目前阅读了grpahscope的官方文档，了解到graphscope内置了networkx接口，但从文档的描述中发现graohscope只内置了部分networkx的算法接口，并没有内置networkx中一些常见的算法接口如介数中心性、紧密中心性算法等。如果我在开发中需要使用的这些未内置的算法，请问需要如何操作？
graphscope是否集成networkx的所有算法

**Comment by sighingnow** [comment]
@acezen Could you please give some comment about the question above?

Thanks!

**Reply by JackyYangPassion** [reply]
I think that in graph data mining in a production environment, analysts mainly perform graph mining analysis through networkx. When the amount of data increases, a distributed algorithm computing engine is needed to support it, such as GAE. If there is a corresponding implementation, the adaptation will be faster.
